src/AutoRest.CSharp/Common/Generation/Writers/CodeWriter.cs

```diff
@@ -635,11 +635,11 @@ public string ToString(bool header)
                 return string.Empty;
             }
 
-            var builder = new StringBuilder();
+            var builder = new StringBuilder(_length);
             string[] namespaces = _usingNamespaces
                     .Distinct()
```

Comment:
It looks like this list of namespaces is only used to add and then here to order.  If we change it to a HashSet<string> instead of List<string> we can avoid the Distinct call

Reply:
fixed.